tests/turo/Makefile

```diff
@@ -0,0 +1,8 @@
+include ../Makefile.tests_common
+
+ifndef MODULE_TEST_UTILS_RESULT_OUTPUT
+  OUTPUT_FORMAT ?= check
+  USEMODULE += test_utils_result_output_${OUTPUT_FORMAT}
+endif
```

Comment:
I'm not sure this `ifndef` is needed? If `TEST_KCONFIG=1` then `USEMODULE` is just overridden.